lib/beaker-hostgenerator/data.rb

```diff
@@ -398,55 +422,62 @@ def osinfo
         },
         'opensuse11-32' => {
```

Comment:
We don't do anything at all with opensuse do we?

Reply:
@shrug who should we direct this question towards? I assume someone like @joshcooper would be able to answer it, but if he doesn't have time for this, who would be a good person to ping to get this information?

Reply:
In order to move forward with this, let's just remove `packaging_platform` from the `opensuse` test targets. It sounds like I was wrong in my assumption the sles packages would just work for opensuse. If this becomes an issue in the future, we can figure out a better solution.